### DISCUSSION TITLE
Facility, paradrop capable, which column?

### DISCUSSION THREAD

**Discussion by claudejdev** [PygmalionOfCyprus/cmo-db-requests#534] [Q&A] [answered]
Hey there,
In which column, of which table, is a facility set paradrop capable or not?

I'm contemplating writing down a request and I would like it to encompass all units affected, thus I want to query the dB first.

All the best

**Comment by PygmalionOfCyprus** [comment, marked as answer]
My bad for not noticing this. For some reason GitHub doesn't want to notify me when people post new things in the Discussions tab.

I assume you're asking about mounts and not facilities? Boolean Cargo_ParadropCapable in DataMount.

**Reply by claudejdev** [reply]
Thanks mate. No sweat, there was no rush, I'm glad you answered, that clears my doubts.
